lib/knapsack_pro/report.rb

```diff
@@ -12,7 +12,6 @@ def self.save
 
     def self.save_subset_queue_to_file
       test_files = KnapsackPro.tracker.to_a
-      KnapsackPro.tracker.reset!
```

Comment:
It was not that good place to reset tracker. `save_subset_queue_to_file` should only read from tracker.